### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Robster100** [openaudible/openaudible#1032] [Ideas]
Currently my 400 audio books are taking over 600Gb of disk space!

Having converted the aax file to m4b why does OpenAudible have to keep the original aax file around, could there not be a setting to automatically delete the original aax file?

Surely it would not take much to have a small database of all the downloaded aax files with the critical information to decide if a new version has been published and to initiate a new download/conversion process?

Duplicating the files, in fact triplicating the files if you create a web page is such a huge waste of space to almost make OpenAuidible unusable, or a 'one time use' tool?

But I love the idea of being able to 'checkin' with Audible every now and again to make sure I have all my books backed up and available as very useful, so would love to see this waste of space issue resolved!

**Comment by openaudible** [comment]
I am adding a feature to allow automatic delete of aax files. Will post a message here with a beta is available.

Also working on combining the web and library to avoid duplicates. But it might make sense to use something like [audiobookshelf ](https://www.audiobookshelf.org/) for a real nice web interface.

**Reply by Robster100** [reply]
That is excellent news, many thanks for the response.

In the meantime, would there be any detrimental affect to Openaudible and its operation if I manually deleted the aax files?

I don't want it to start downloading 400 plus audio books again!

**Reply by openaudible** [reply]
It depends on the version.. At some point we added a check to not download books if you already have a converted version.
So you could delete 1 or two... and see if they show up as needing to be downloaded.

**Comment by openaudible** [comment]
Ability to delete the aax files automatically has been added to beta 3.6.1.1.
https://openaudible.org/beta
